datahub-web-react/src/graphql/scroll.graphql

```diff
@@ -0,0 +1,399 @@
+fragment downloadSearchResults on Entity {
```

Comment:
Could we instead add another subdirectory called scroll? Imo this makes more sense to be in a path like `scroll/search.graphql`

Reply:
Why is that? This directory is pretty flat already and i am not sure that this warrants a change

Reply:
I was actually considering putting this in either a) search.graphql or in a new b) download_csv.graphql